packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -526,85 +503,383 @@ export const Tabs = (props: TabsProps) => {
     }
   };
 
-  const createTab = React.useCallback(
-    (label: React.ReactNode, index: number) => {
-      const onClick = () => {
-        setFocusedIndex(index);
-        onTabClick(index);
-      };
-      return (
-        <li key={index}>
-          {!React.isValidElement(label) ? (
-            <Tab
-              label={label}
-              className={cx({
-                'iui-active': index === currentActiveIndex,
-              })}
-              tabIndex={index === currentActiveIndex ? 0 : -1}
-              onClick={onClick}
-              aria-selected={index === currentActiveIndex}
-            />
-          ) : (
-            React.cloneElement(label as JSX.Element, {
-              active: index === currentActiveIndex,
-              'aria-selected': index === currentActiveIndex,
-              tabIndex: index === currentActiveIndex ? 0 : -1,
-              onClick: (args: unknown) => {
-                onClick();
-                label.props.onClick?.(args);
-              },
-            })
-          )}
-        </li>
-      );
-    },
-    [currentActiveIndex, onTabClick],
+  return (
+    <Box
+      as='ul'
+      className={cx(
+        'iui-tabs',
+        `iui-${type}`,
+        {
+          'iui-green': color === 'green',
+          'iui-animated': type !== 'default' && isClient,
+          'iui-not-animated': type !== 'default' && !isClient,
+          'iui-large': hasSublabel,
+        },
+        className,
+      )}
+      data-iui-overflow={overflowOptions?.useOverflow}
+      data-iui-scroll-placement={scrollingPlacement}
+      role='tablist'
+      onKeyDown={onKeyDown}
+      ref={refs}
+      {...rest}
+    >
+      {items.map((item, index) => {
+        return (
+          <li key={index}>
```

Comment:
do we need to use `<ul>`/`<li>` elements? i think divs should work just fine, and we'll be able to remove this extra element completely.

```diff
- <ul role=tablist>
+ <div role=tablist>
-   <li>
      <button role=tab>...</button>
-   </li>
- </ul>
+ </div>
```

Reply:
The sizing for the vertical tabs is messed up when the `li` wrapper is removed.

![image](https://github.com/iTwin/iTwinUI/assets/78564221/1ca73349-9e60-4221-a262-e3b1390081ca)

This is the only part of the tabs css that targets `li` elements, but the same properties applied to it are also applied to `iui-tab`

![image](https://github.com/iTwin/iTwinUI/assets/78564221/2c24f7ad-c255-4edb-8132-34b77dfacdaa)

Reply:
have you tried investigating why and fixing it? maybe there's an `li` somewhere else, or it relies on some parent-child relationship

Reply:
The `ul` element has been replaced with `div` and the `li` wrapper has been removed